src/model.ts

```diff
@@ -106,9 +140,26 @@ export class Model {
           tag = obj.toString();
           break;
         case 'blob':
-          // blob = obj;
           blob = Buffer.from(obj);
           break;
+        case 'batchsize':
+          batchsize = parseInt(obj.toString(), 10);
+          break;
+        case 'minbatchsize':
+          minbatchsize = parseInt(obj.toString(), 10);
+          break;
+        case 'inputs':
+          // tslint:disable-next-line:prefer-for-of
+          for (let j = 0; j < obj.length; j++) {
+            inputs.push(obj[j].toString());
+          }
```

Comment:
obj.forEach(input =>  inputs.push(input.toString()))

Reply:
solved in the lastest commit